src/site/content/en/metrics/inp/index.md

```diff
@@ -110,14 +110,19 @@ Since INP is experimental, the guidance around thresholds may change over time a
 
 The primary driver of interactivity is often JavaScript, though browsers do provide interactivity through controls _not_ powered by JavaScript, such as checkboxes, radio buttons, and controls powered by CSS.
 
-As far as INP goes, **only the following interaction types are observed:**
+As far as INP goes, **only the following _synchronous_ interaction types are observed:**
 
 - Clicking with a mouse.
 - Tapping on a device with a touchscreen.
 - Pressing a key on either a physical or onscreen keyboard.
 
 {% Aside 'important' %}
+<p>
 Hovering and scrolling does not factor into INP. However, scrolling with the keyboard (space bar, page up, page down, and so forth) involves a keystroke, which may trigger other events that INP _does_ measure. Any resulting scrolling is _not_ factored into how INP is calculated.
+</p>
+<p>
+Additionally, only synchronous work is considered by INP. Asynchronous work from a previous interaction _may_ factor into the latency observed in a later interaction if the result of that asynchronous work kicks off a task that blocks the main thread.
```

Comment:
This looks good to me, but would love to hear @mmocny's thoughts as well. It's a holiday in Canada so may need to wait till tomorrow.

Reply:
@mmocny WDYT?

Reply:
(thanks for ping, I forgot to review this patch)

I... worry that we are trying to add clarifications in ways that actually makes it less clear.  Sorry :(

How should I interpret that "**only synchronous work** is considered for INP" but also "Asynchronous work [...] may affect INP"?

Also it's not just "async work _from a previous interaction_"... it's **any long tasks** on main regardless where they come from.

There is a nuanced point to make that if you move code that used to run sync inside the handlers of an Event and was blocking next frame, by scheduling it to run in a future task after frame... it will still be bad for performance in the long run -- but we can just make that point directly: **INP is affected by long tasks**.  Shuffling long tasks around can help sometimes but is still problematic.

Scott H recently put together a very good document explaining how scheduling affects INP and I'll share that in case it helps.

---

Here's what I think I would suggest: we may need to **double down** on explaining what it means to measure **Interaction** to **Next Paint** so folks understand it so intuitively, they don't need clarifications.  If folks ("really") understood that concept I think these nuanced questions would become more obvious.  My 2cents.

Perhaps that means reminding folks about [rendering architecture](https://developer.chrome.com/articles/renderingng/), or [the event loop](https://www.youtube.com/watch?v=cCOL7MC4Pl0), or explaining the life of a frame in intuitive terms by using an interactive demo like Annie's cookie clicker.

Philip's Optimizing LCP presentation from Google I/O, which breaks down the time that blocks LCP into intuitive segments, I think is a good aspirational goal.  But it took years to get to the point where we had that level of intuition, so it may just take time?

---

Besides just understanding what INP measures at a more intuitive level, I think some things folks are specifically confused about INP definition:

- When there is no need for a new paint (i.e. no visual update) folks assume we will wait and measure until some eventual visual update: 
  - Maybe that's a misunderstanding of the term "Paint" which we need to explain?

- When the Next Paint is "unrelated" to the interaction itself, folks don't consider the Interaction "complete", and so also assume we will keep measuring:
  - Maybe we explain that INP measures the effects of the interaction on the page performance overall, not on the quality/semantic-value of the feedback to the interaction itself.

Reply:
I think changing the focus to long tasks is a good first step, but do you think this would benefit from a short explanation about the event loop? Nothing hairy like we did in the watch party today, but a simple explanation that explains _why_ long tasks block? Or is this too much background information for what could be a more direct explanation?